src/main/scala/com/github/unchama/seichiassist/subsystems/gacha/bukkit/actions/BukkitDrawGacha.scala

```diff
@@ -0,0 +1,122 @@
+              case _ =>
+                ""
+            }
+
+            GachaRarity.of(gachaPrize) match {
+              case GachaRarity.Gigantic =>
+                val prizeItem = gachaPrize.itemStack
+
+                val localizedEnchantmentList =
+                  prizeItem.getItemMeta.getEnchants.asScala.toSeq.map {
+                    case (enchantment, level) =>
+                      s"$GRAY${EnchantNameToJapanese.getEnchantName(enchantment.getName, level)}"
+                  }
+
+                import scala.util.chaining._
+                val message =
+                  new TextComponent().tap { c =>
+                    import c._
+                    setText(
+                      s"$AQUA${prizeItem.getItemMeta.getDisplayName}${GOLD}を引きました！おめでとうございます！"
+                    )
+                    setHoverEvent {
+                      new HoverEvent(
+                        HoverEvent.Action.SHOW_TEXT,
+                        Array(
+                          new TextComponent(
+                            s" ${prizeItem.getItemMeta.getDisplayName}\n" +
+                              ListFormatters.getDescFormat(localizedEnchantmentList.toList) +
+                              ListFormatters
+                                .getDescFormat(prizeItem.getItemMeta.getLore.asScala.toList)
+                          )
+                        )
+                      )
+                    }
+                  }
+                Sync[F].delay {
+                  player.sendMessage(s"${RED}おめでとう！！！！！Gigantic☆大当たり！$additionalMessage")
+                  player.spigot().sendMessage(message)
+                  sendMessageToEveryone(s"$GOLD${player.getName}がガチャでGigantic☆大当たり！")(
+                    forString[IO]
+                  )
+                  sendMessageToEveryone(message)(forTextComponent[IO])
+                  SendSoundEffect.sendEverySoundWithoutIgnore(
+                    Sound.ENTITY_ENDERDRAGON_DEATH,
+                    0.5f,
+                    2f
+                  )
+                }
+              case GachaRarity.Big =>
+                Sync[F].delay {
+                  player.playSound(player.getLocation, Sound.ENTITY_WITHER_SPAWN, 0.8f, 1f)
+                  if (count == 1) player.sendMessage(s"${GOLD}おめでとう！！大当たり！$additionalMessage")
+                }
+              case GachaRarity.Regular if count == 1 =>
+                Sync[F].delay {
+                  player.sendMessage(s"${YELLOW}おめでとう！当たり！$additionalMessage")
+                }
+              case GachaRarity.GachaRingoOrExpBottle if count == 1 =>
+                Sync[F].delay {
+                  player.sendMessage(s"${WHITE}はずれ！また遊んでね！$additionalMessage")
+                }
+              case _ => Sync[F].unit
+            }
+
+            if (count > 1) {
+              Sync[F].delay {
+                player.sendMessage(s"$AQUA${count}回ガチャを回しました。")
+              }
+            } else Sync[F].unit
```

Comment:
一個上の作用が実行されていません（作用が二つに分かれていて、二つ目しか利用されていない）

```suggestion
      _ <-
        if (count > 1) {
          Sync[F].delay {
            player.sendMessage(s"$AQUA${count}回ガチャを回しました。")
          }
        } else Sync[F].unit
```